### PR TITLE
Azure Maps Traffic SDK

### PR DESCRIPTION
**Pull request by khmic5**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/maps/azure-maps-traffic/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-08-19)
```
Comment (dubiety) [inline sdk/maps/azure-maps-traffic/CHANGELOG.md:3]: Please correct the release date. The content also wrong. Should be "Azure Maps Traffic" client library for Java

File: sdk/maps/azure-maps-traffic/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+The sample code assumes global Azure. Please change `AzureEnvironment.AZURE` variable if otherwise.
+
+See [Authentication][authenticate] for more options.
+
+## Key concepts
+
+See [API design][design] for general introduction on design and key concepts on Azure Management Libraries.
+
+## Examples
+
```
Comment (dubiety) [inline sdk/maps/azure-maps-traffic/README.md:76]: Where's the examples?

File: sdk/maps/azure-maps-traffic/README.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    new TrafficIncidentViewportOptions()
+        .setBoundingBox(new GeoBoundingBox(45, 45, 45, 45))
+        .setBoundingZoom(2).setOverview(new GeoBoundingBox(45, 45, 45, 45))
+        .setOverviewZoom(2).setCopyright(true));
+```
+
+## Troubleshooting
+When you interact with the Azure Maps Services, errors returned by the Maps service correspond to the same HTTP status codes returned for REST API requests.
+
+For example, if you search with an invalid coordinate, a error is returned, indicating "Bad Request".400
```
Comment (lmolkova) [inline sdk/maps/azure-maps-traffic/README.md:171]:
```suggestion
For example, if you search with an invalid coordinate, a error is returned, indicating `400 - Bad Request`
```

**Review by lmolkova — APPROVED** [review]

**Pull request merged**